spec/spec_helper.rb

```diff
@@ -18,6 +18,7 @@
 
 Aws.config.update region: 'us-east-1', credentials: Aws::Credentials.new('', '')
 
+
```

Comment:
Remove extra spacing added. This way the PR only shows files with relevant changes.

Reply:
Done